src/app/page.css

```
@@ -1,1002 +1,1164 @@
 /*@layer components {*/
```

Stylelint (17.3.0)

[error] Expected whitespace after "/*" (comment-whitespace-inside)

(comment-whitespace-inside)